client/src/components/signup/signup.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import '../../containers/main.css';
+import '../common/navbar.js';
+
+export default class Signup extends Component {
+    render() {
+        return(
+            <html>
+                <body>
+                    <div className="container signup-box">
+                        <h1>Sign up for MegaminerAI 2017!</h1>
```

Comment:
It would be really cool to replace "MegaminerAI 2017" with a prop that is called `competitionName` and allow different names to be passed to it. Let me know if you need help doing that.